cumulusci/core/tasks.py

```diff
@@ -9,11 +9,13 @@
 from past.utils import old_div
 import logging
 import time
+import threading
 
 from cumulusci.core.exceptions import TaskRequiresSalesforceOrg
 from cumulusci.core.exceptions import TaskOptionsError
 
-CURRENT_TASK = None
+CURRENT_TASK = threading.local()
+CURRENT_TASK.instance = None
```

Comment:
Just to confirm, we don't have any other packages using this with the assumption that it's a task rather than an object with an instance member, right?